docs/source/topics/con_about-usage-data-collection.adoc

```diff
@@ -0,0 +1,12 @@
+[id="about-usage-data-collection_{context}"]
+= About usage data collection
+
+{prod} prompts you before use for optional, anonymous usage data collection to assist with development.
+No personally identifiable information is collected.
+Consent for usage data collection can be granted or revoked by you at any time.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about collected data, see the Red{nbsp}Hat link:https://developers.redhat.com/article/tool-data-collection[Telemetry data collection notice].
```

Comment:
{nbsp} :-P

Reply:
`Red{nbsp}Hat` => ` Red Hat` need to updated, otherwise looks good to me.

Reply:
This could be replaced with `{rh}` (which will expand to `Red{nbsp}Hat` https://github.com/code-ready/crc/blob/0691cff8afd7656b20b46448e263b3f22fcc1784/docs/source/meta/product-attributes.adoc#L13 - I'm assuming this stands for 'non-breaking space', which makes sense here)

Reply:
Yes, the `{nbsp}` is a non-breaking space. Good idea replacing it with the already-defined attribute. :)

Reply:
Updated and replaced with the attribute in both locations.